pom.xml

```diff
@@ -304,9 +304,9 @@
         <ballerina.version>1.2.39</ballerina.version>
         <ballerina.jre.artifacts.version>2.0.1</ballerina.jre.artifacts.version>
         <project.scm.id>ballerina-scm-server</project.scm.id>
-        <docker.version>1.2.39</docker.version>
-        <kubernetes.version>1.2.39</kubernetes.version>
-        <awslambda.version>1.2.39</awslambda.version>
+        <docker.version>1.2.38</docker.version>
```

Comment:
Why are we downgrading these? Didn't we release them

Reply:
it is just for this PR will update when the ara available on nexus

Reply:
We have to release them first and then update the distribution. That way it would be easier right? No need to create 2 PRs

Reply:
Done